src/content/docs/fr/guides/migrate-to-astro.mdx

```diff
@@ -25,7 +25,7 @@ Selon votre projet déjà existant, vous pourrez peut-être utiliser vos composa
 
 - [Feuilles de style ou bibliothèques CSS](/fr/guides/styling/) incluant Tailwind.
 
-- [Fichiers Markdown/MDX](/fr/guides/markdown-content/), configuré en utilisant votre [plugin remark et rehype](/fr/guides/markdown-content/#configuration-de-markdown-et-mdx).
+- [Fichiers Markdown/MDX](/fr/guides/markdown-content/), configuré en utilisant votre [plugin remark et rehype](/fr/guides/markdown-content/#configuration-de-markdown).
```

Comment:
The anchor seems wrong: see [`l28`·](https://github.com/withastro/docs/blob/7879a1f4c155cb6b357a64cacf6e26fb8d7aa735/src/content/docs/en/guides/migrate-to-astro.mdx?plain=1#L28) I think this should be:
```suggestion
- [Fichiers Markdown/MDX](/fr/guides/markdown-content/), configuré en utilisant votre [plugin remark et rehype](/fr/guides/markdown-content/#plugins-markdown).
```